app/builders/messages/instagram/base_message_builder.rb

```diff
@@ -94,11 +94,10 @@ def message_reply_attributes
   def build_message
     # Duplicate webhook events may be sent for the same message
     # when a user is connected to the Instagram account through both Messenger and Instagram login.
+    # There is chance for echo events to be sent for the same message.
     # Therefore, we need to check if the message already exists before creating it.
     return if message_already_exists?
 
-    return if @outgoing_echo
```

Comment:
We don't need this condition since we are already checking if the message exists or not.